tests/unit/test_api_request.py

```diff
@@ -109,6 +109,8 @@ def test_build_filter_header(self, list_operation):
             SimpleNamespace(
                 filterable_result="bar",
                 filterable_list_result=["foo", "bar"],
+                order_by=None,
+                order=None,
```

Comment:
Could we add some extra tests to check the generated header with values specified for order and order_by?